src/scenicplus/cistromes.py

```diff
@@ -314,6 +314,93 @@ def TF_cistrome_correlation(scplus_obj: SCENICPLUS,
         scplus_obj.uns['TF_cistrome_correlation'][out_key] = {}
     scplus_obj.uns['TF_cistrome_correlation'][out_key] = corr_df
 
+def cistrome_correlation(scplus_obj: SCENICPLUS,
+                         variable: str = None,
+                         use_pseudobulk: bool = True,
+                         auc_key: str = 'Cistromes_AUC',
+                         signature_key1: str = 'Gene_based',
+                         signature_key2: str = 'Region_based',
```

Comment:
I left in the option to specify the signature names for flexibility, although I imagine that the defaults will be sufficient most of the time